src/steps/users/contact-field-equals.ts

```diff
@@ -1,81 +1,64 @@
-/*tslint:disable:no-else-after-return*/
-/*tslint:disable:triple-equals*/
-
 import { BaseStep, Field, StepInterface } from '../../core/base-step';
 import { FieldDefinition, RunStepResponse, Step, StepDefinition } from '../../proto/cog_pb';
 
-import { baseOperators } from './../../client/constants/operators';
+import { baseOperators } from '../../client/constants/operators';
 import * as util from '@run-crank/utilities';
 
 /**
  * Note: the class name here becomes this step's stepId.
- * @see BaseStep.getId()
+ @see BaseStep.getId()
  */
-export class UserFieldEqualsStep extends BaseStep implements StepInterface {
+export class ContactFieldEquals extends BaseStep implements StepInterface {
 
-  /**
-   * The name of this step: used when identifying this step to humans.
-   */
-  protected stepName: string = 'Check a field on a JSON Placeholder user';
+  protected stepName: string = 'Check a field on an Iterable Contact';
 
-  /**
-   * Type of step (either Action or Validation).
-   */
   protected stepType: StepDefinition.Type = StepDefinition.Type.VALIDATION;
 
-  /**
-   * A string that can be evaluated as an ECMAScript-compatible regular expression. This is used to
-   * identify and evaluate this step in cucumber-like scenario files. You are encouraged to use
-   * named regex capturing groups that correspond to the expected fields below.
-   */
-  // tslint:disable-next-line:max-line-length
-  protected stepExpression: string = 'the (?<field>.+) field on JSON Placeholder user (?<email>.+) should (?<operator>be less than|be greater than|be|contain|not be|not contain) (?<expectedValue>.+)';
+  protected stepExpression: string = 'the (?<field>[a-zA-Z0-9_]+) field on iterable contact (?<email>.+) should (?<operator>be less than|be greater than|be|contain|not be|not contain) (?<expectedValue>.+)';
 
-  /**
-   * An array of Fields that this step expects to be passed via step data. The value of "field"
-   * will be used as the field value's key when passed over the step data Struct.
-   */
   protected expectedFields: Field[] = [{
     field: 'email',
     type: FieldDefinition.Type.EMAIL,
-    description: "User's email address",
-  }, {
+    description: 'the email address of the contact',
```

Comment:
- [ ] Nit: Recommend keeping a compact `Contact's email address` here.

And below and in other steps.  Generally go for:
- The most compact, but understandable descriptions as you can.
- Sentence case.